Update FAQ for custom objects clarification over associations (#4061)
segmentio/segment-docs: src/connections/destinations/catalog/actions-hubspot-cloud/index.md

## FAQ & Troubleshooting

### How do I send other standard objects to HubSpot?

> Segment provides prebuilt mappings for contacts and companies. If there are other standard objects you would like to create records in, please use the **Create Custom Object Record** action. For example, to create a deal in HubSpot, add a mapping for Create Custom Object Record, set up your Event Trigger criteria, and input a literal string of "deals" as the Object Type. You can use the Properties object to add fields that are in the [deals object](https://developers.hubspot.com/docs/api/crm/deals){:target="_blank"}, such as `dealname` and `dealstage`. The same can be done with other object types (for example, tickets, quotes, etc). For the moment, ending fields that are meant to go to HubSpot outside of the properties object is not supported. This includes sending [associations](https://developers.hubspot.com/docs/api/crm/associations).  Please note, Segment only supports creating new records in these cases; updates to existing records are only supported for contacts and companies. 

Reviewer: ```suggestion
Segment provides prebuilt mappings for contacts and companies. If there are other standard objects you would like to create records in, please use the **Create Custom Object Record** action. For example, to create a deal in HubSpot, add a mapping for Create Custom Object Record, set up your Event Trigger criteria, and input a literal string of "deals" as the Object Type. You can use the Properties object to add fields that are in the [deals object](https://developers.hubspot.com/docs/api/crm/deals){:target="_blank"}, such as `dealname` and `dealstage`. The same can be done with other object types (for example, tickets, quotes, etc). Ending fields that are to go to HubSpot outside of the properties object isn't supported. This includes sending [associations](https://developers.hubspot.com/docs/api/crm/associations){:target="_blank"}.  Please note, Segment only supports creating new records in these cases; updates to existing records are only supported for contacts and companies. 
```

Author: i guess i must misspelled here:
Ending fields that are to go to HubSpot outside of the properties object isn't supported.
